.github/workflows/create-release.yml

```diff
@@ -128,29 +131,34 @@ jobs:
     # Only create the release if workflow is run manually. (This allows the other steps in the workflow to test PRs.)
     if: github.event_name == 'workflow_dispatch'
     steps:
+    # The GitHub Actions bot email was taken from: https://github.community/t/github-actions-bot-email-address/17204/6
+    - name: Set bot user data for commits
+      run: |
+        git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
+        git config --global user.name "GitHub Actions Bot"
 
     - name: Checkout spinalcordtoolbox (main branch)
       uses: actions/checkout@v2
       with:
         ref: ${{ env.MAIN_BRANCH }}
         fetch-depth: 0
 
-    - name: Download requirements-freeze.txt
+    - name: Update version.txt (main branch)
+      run: | 
+          echo "${{ github.event.inputs.future_milestone_title }}.dev0" > spinalcordtoolbox/version.txt
+          git commit -m "Update version.txt for ${{ github.event.inputs.future_milestone_title }} development"
```

Comment:
Missing `git add`:
```suggestion
          echo "${{ github.event.inputs.future_milestone_title }}.dev0" > spinalcordtoolbox/version.txt
          git add spinalcordtoolbox/version.txt
          git commit -m "Update version.txt for ${{ github.event.inputs.future_milestone_title }} development"
```

Reply:
Good catch! (Both with the `git add` part and the above branch structure part.)